python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -186,6 +182,10 @@ def run(self, rawRef, calexpRef, templateIds=None, reuse=[]):
             - differencer : output of `config.differencer.run` (`lsst.pipe.base.Struct` or `None`).
             - associator : output of `config.associator.run` (`lsst.pipe.base.Struct` or `None`).
         """
+        if reuse is None:
+            reuse = []
```

Comment:
Good catch and fix. Though you should document the None choice in the docstring.